MJ68-sleep.ict

```diff
@@ -478,13 +478,60 @@ rooms:
       this.arm.rotation = 170;
       this.arm.scale.x = 0.5;
       this.won = false;
+      this.mosquitoCounter = 0;
+      this.maxMosquito = 5;
+
+      // Returns a mosquito that is set to spawn along the perimeter of the room
+      this.SpawnMosquito = (() => {
+          var edgeSpawn = Math.floor(ct.random.range(0, 4)); // Random screen edge for the spawn
+
+          var maxWidth = 800; // TODO - get this value from properties
+          var maxHeight = 600; // TODO - get this value from properties
```

Comment:
Use `ct.camera.width` and `ct.camera.height`

Reply:
Fixed in [d038378](https://github.com/annedo/Minijam68-sleep/pull/2/commits/d038378399a1b607341f7a75908fdaab5217cb83)